ftplugin/go.vim

```diff
@@ -74,9 +74,8 @@ if get(g:, "go_textobj_enabled", 1)
   xnoremap <buffer> <silent> [[ :<c-u>call go#textobj#FunctionJump('v', 'prev')<cr>
 endif
 
-if go#config#AutoTypeInfo() || go#config#AutoSameids()
-  let &l:updatetime= get(g:, "go_updatetime", 800)
-endif
+" Set up autocmd to call :GoSameIds / :GoInfo automatically
+call go#auto#update_autocmd()
```

Comment:
Given that there's a CursorHold autocmd that calls `go#auto#update_autocmd`, is this call necessary?

Reply:
Not exactly because `BufEnter` does. I removed this line: 3b576be.